Sources/PortalSwift/Portal.swift

```diff
@@ -101,7 +101,7 @@ public class Portal {
 
     // Creating this as a variable first so it's usable to
     // fetch the client in the Task at the end of the initializer
-    let api = api ?? PortalApi(apiKey: apiKey, apiHost: apiHost, provider: provider)
+    let api = api ?? PortalApi(apiKey: apiKey, apiHost: apiHost, provider: self.provider)
```

Comment:
i think you dont have the correct swift format settings. Can you ask blake about how to get that set up properly for your development environment.

Reply:
thats why its adding a self everywhere.